code/datums/components/_component.dm

```diff
@@ -583,4 +583,4 @@
 //for being appended to tooltips.    //Never iterate over all components on an item when making a tooltip, at least until someone refactors my code -Tarmunora
 /datum/component/proc/getTooltipDesc()
 	SHOULD_CALL_PARENT(1)
-	return list()
\ No newline at end of file
+	return list()
```

Comment:
Out of the scope of this PR but this should get axed in favor of using the new `COMSIG_ITEM_BUILD_TOOLTIP` signal.